content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,137 @@
+
+{{< caution >}}
+
+Make sure to back up your certificate directory along with configuration files and any other necessary files.
+
+This approach assumes operation of the Kubernetes control plane in a HA configuration with multiple API servers. Graceful termination of the API server is also assumed so clients can cleanly disconnect from one API server and reconnect to another. 
+
+Configurations with a single API server will experience unavailability while the API server is being restarted.
+
+{{< /caution >}}
+    
+1. Distribute new CA private keys (`ca.key` and `front-proxy-ca.key`) to all your control plane nodes in the Kubernetes certificates directory.
+
+1. Update *Kubernetes controller manager's* `--root-ca-file` to include both old and new CA and restart controller manager.
+
+    Any service account created after this point will get secrets that include both old and new CAs.
+
+    {{< note >}}
+
+    Remove the flag `--client-ca-file` from the *Kubernetes controller manager* configuration. You can also replace the existing client CA file or change this configuration item to reference a new, updated CA. [Issue 1350](https://github.com/kubernetes/kubeadm/issues/1350) tracks an issue with *Kubernetes controller manager* being unable to accept a CA bundle.
+
+    {{< /note >}}
+
+1. Update all service account tokens to include both old + new CA certificates.
+    
+    If any pods started before new CA is picked by API servers, they will get this update and trust both old + new CAs.
+
+    ```shell
+    base64_encoded_ca="$(base64 <path to file containing both old + new CAs>)"
+
+    for namespace in $(kubectl get ns --no-headers | awk '{print $1}'); do
+        for token in $(kubectl get secrets --namespace "$namespace" --field-selector type=kubernetes.io/service-account-token -o name); do
+            kubectl get $token --namespace "$namespace" -o yaml | \
+            /bin/sed "s/\(ca.crt:\).*/\1 ${base64_encoded_ca}" | \
+            kubectl apply -f -
+        done
+    done
+    ```
+
+1. Restart all pods using in-cluster configs (ex: kube-proxy, coredns, etc) so they can use the updated certificate authority data from *ServiceAccount* secrets.
+
+    *   Make sure coredns, kube-proxy and other pods using in-cluster configs are working as expected.
+
+1. Append new CA to the file against `--client-ca-file` flag in the `kube-apiserver` and `kube-scheduler` configuration.
+
+    {{< note >}}
+
+    If you used `kubeadm` to deploy your cluster, the control plane components run as {{< glossary_tooltip text="static Pods" term_id="static-pod" >}}, managed by the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}.
+
+    {{< /note >}}
+
+1. Update certificates for user accounts by replacing the content of `client-certificate-data` and `client-key-data` respectively and the `certificate-authority-data` section in all `admin.conf`, `controller-manager.conf`, `scheduler.conf` files with Base64-encoded new certificate data:
```

Comment:
I'd suggest splitting this step in to two parts. A single step conflates users' kubeconfigs with kubeadm configuration

Additionally, users will need to update the `certificate-authority-data` or `certificate-authority` to include both CAs in their kubeconfigs